### PR TITLE
Exfiltrate Linux Logs With Dropbox

### PR DESCRIPTION
**Pull request by aleff-github**

### CODE REVIEW

**Comment by dallaswinger** [comment]
Exfiltrate is probably a "more appropriate" word than "Steal"

especially considering this is in the exfiltration category :)

**Comment by aleff-github** [comment]
> Exfiltrate is probably a "more appropriate" word than "Steal"
> 
> especially considering this is in the exfiltration category :)

thank you for telling me, I agree

**Comment by dallaswinger** [comment]
> > Exfiltrate is probably a "more appropriate" word than "Steal"
> > especially considering this is in the exfiltration category :)
> 
> thank you for telling me, I agree

Please replace references within the files as well <3

**Comment by aleff-github** [comment]
> Please replace references within the files as well <3

Yes sorry, I had forgotten

File: payloads/library/exfiltration/Steal Linux Log Files/payload.txt [hunk 1]
```diff
@@ -0,0 +1,80 @@
+REM ######################################################
```
Comment (dallaswinger) [inline payloads/library/exfiltration/Steal Linux Log Files/payload.txt:1]: old file from before rename?

**Pull request merged**